manual/speculative_execution/README.md

```diff
@@ -66,15 +66,15 @@ there is no way to guarantee that only one node will apply the mutation.
 ### Enabling speculative executions
 
 Speculative executions are controlled by an instance of
-[SpeculativeExecutionPolicy][sep] provided when initializing the
+[SpeculativeExecutionPolicy] provided when initializing the
```

Comment:
Not entirely related to the changes here, but wanted to point out we use the term 'cancel' a few times, i.e.:

```
If that second node replies faster, we can send the response back to the client (we also cancel the first query)
```

We should clarify that by cancel we mean that we disregard/throw away any subsequent response received.  In the past people have perceived `ResultSetFuture.cancel()` as possibly cancelling the request, which it's really just discarding any response right?